<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cvs::Internal::SettingsPage</class>
 <widget class="QWidget" name="Cvs::Internal::SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>281</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="commandLabel">
        <property name="text">
         <string>CVS command:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::PathChooser" name="commandPathChooser" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="rootLabel">
        <property name="text">
         <string>CVS root:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="rootLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="miscGroupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="diffOptionsLabel">
        <property name="text">
         <string>Diff options:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="diffOptionsLineEdit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="promptToSubmitCheckBox">
        <property name="text">
         <string>Prompt on submit</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="describeByCommitIdCheckBox">
        <property name="toolTip">
         <string>When checked, all files touched by a commit will be displayed when clicking on a revision number in the annotation view (retrieved via commit ID). Otherwise, only the respective file will be displayed.</string>
        </property>
        <property name="text">
         <string>Describe all files matching commit id</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="timeOutLabel">
        <property name="text">
         <string>Timeout:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="timeOutSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rootLineEdit</tabstop>
  <tabstop>timeOutSpinBox</tabstop>
  <tabstop>diffOptionsLineEdit</tabstop>
  <tabstop>promptToSubmitCheckBox</tabstop>
  <tabstop>describeByCommitIdCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
